src/client.ts

```diff
@@ -3795,29 +3795,38 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
     }
 
     /**
-     * Set a user's power level.
+     * Set a power level to one or multiple users.
      * @param {string} roomId
-     * @param {string} userId
+     * @param {string|string[]} userId
      * @param {Number} powerLevel
      * @param {MatrixEvent} event
      * @return {Promise} Resolves: to an ISendEventResponse object
      * @return {module:http-api.MatrixError} Rejects: with an error response.
      */
     public setPowerLevel(
         roomId: string,
-        userId: string,
+        userId: string | string[],
         powerLevel: number,
         event: MatrixEvent,
     ): Promise<ISendEventResponse> {
-        let content = {
+        type EventContent = {
```

Comment:
the utils.deepCopy later on breaks this type, given it'll include other fields like `events`, `state_default`, etc. Once the typing for `deepCopy` is fixed this would no longer comply.

Reply:
I see, any idea how to satisfy the strict type checker? I don't get why it fails. Locally it does not have any TypeScript issues in strict mode.

Reply:
Make sure to also run `--noImplicitAny` given we (wrongly) specify that as `false` in tsconfig

Reply:
Ah, thanks for the hint 💡  Is this a better solution: https://github.com/matrix-org/matrix-js-sdk/pull/2892/commits/763532f84a5774b90ce62208e8908545ec16ed7c?